### ISSUE TITLE
Exception: 'unibo.it' is not a valid language tag!

### ISSUE THREAD

**Issue by urbanmatthias** [emmo-repo/EMMO#73]
Hello,

I get the following error, when parsing EMMO using [rdflib](https://rdflib.readthedocs.io/en/stable/):

```
  File "/home/urba/miniconda3/envs/osp/lib/python3.8/site-packages/rdflib/term.py", line 566, in __new__
    raise Exception("'%s' is not a valid language tag!"%lang)
Exception: 'unibo.it' is not a valid language tag!
```

I guess this is the reason:

```xml
<rdfs:comment xml:lang="unibo.it">Contacts:
Gerhard Goldbeck
Goldbeck Consulting Ltd (UK)
email: gerhard@goldbeck-consulting.com

Emanuele Ghedini
University of Bologna (IT)
email: emanuele.ghedini</rdfs:comment>
      
```

**Comment by jesper-friis** [comment]
It should be fixed in the 1.0.0-alpha2 release.

**Comment by jesper-friis** [comment]
It should have been fixed a while ago in 1.0.0-alpha2. I have now updated the inferred ontology available on 
https://raw.githubusercontent.com/emmo-repo/emmo-repo.github.io/master/versions/1.0.0-alpha2/emmo-inferred.owl
to exactly comply with http://emmo.info/emmo/1.0.0-alpha2.

Please confirm that this works, then I will close the issue.

**Comment by urbanmatthias** [comment]
It works, thank you
